### PR TITLE
Added dialects from cloud siem Sumologic

### PR DESCRIPTION
**Pull request by omarrdaniel**
Adding json file (from stix map and to stix map) for the dialect cloud siem Sumologic, following this schema: https://help.sumologic.com/docs/cse/get-started-with-cloud-siem/insight-generation-process/#entities-in-messages-are-mapped-to-entity-type-schema-attributes

### CODE REVIEW

File: stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+  ],
+  "dstDevice_ip": [
+    {
+      "key": "ipv4-addr.value",
+      "object": "dst_ip"
+    },
+    {
+      "key": "network-traffic.dst_ref",
+      "object": "nt",
+      "references": "src_ip"
```
Comment (pcoccoli) [inline stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json]: This should be `dst_ip` right?

File: stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+      "object": "host_mac"
+    }
+  ],
+  "srcDevice_mac": [
+    {
+      "key": "mac-addr.value",
+      "object": "src_mac"
+    },
+    {
+      "key": "ipv4-addr.resolves_to_ref",
```
Comment (pcoccoli) [inline stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json]: Should be plural; i.e. `resolves_to_refs`

File: stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json (hunk 1 above)
Comment (omarrdaniel) [reply stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json]: Yes, fixed it

File: stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json (hunk 2 above)
Comment (omarrdaniel) [reply stix_shifter_modules/sumologic/stix_translation/json/cloud_siem_to_stix_map.json]: Fixed it too, even the one below

**Review by pcoccoli — APPROVED** [review]
Looks good to me!

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**